GTMAppAuthSwift/Sources/GTMAppAuthFetcherAuthorization.swift

```diff
@@ -185,15 +187,18 @@ import GTMSessionFetcher
   /// - Throws: Any error that may arise during removal.
   /// - Note: See `GTMKeychain.swift` for ``GTMAppAuthSwift/GTMKeychainError``s indirectly thrown by
   ///   calls to this method.
-  @available(macOS 10.15, *)
+  @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
```

Comment:
Same as above.

Reply:
Updated to using `if #available` within the implementation.